config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -1099,7 +1099,13 @@ presubmits:
           value: external-dependencies-version vendor vendor-licenses
         image: gcr.io/k8s-testimages/kubekins-e2e:v20200802-ff98b58-1.16
         name: main
-        resources: {}
+        resources:
+          limits:
+            cpu: "2"
```

Comment:
silly nit, but can you drop the quotes in a followup PR? config-forker drops quotes, I'd like to reduce the diff